openmdao/core/group.py

```diff
@@ -3272,7 +3272,11 @@ def _get_auto_ivc_out_val(self, tgts, vars_to_gather, all_abs2meta_in, abs2meta_
                 if val is None:
                     val = value
                 else:
-                    val[:] = value
+                    try:
+                        val[:] = value
+                    except ValueError:
+                        raise ValueError(f"Could not broadcast input array '{tgt}' from shape "
+                                         f"{value.shape} into shape {val.shape}.")
```

Comment:
seems like there are other possible `ValueError` messages that could occur...  
Maybe you should just prefix whatever error comes back, e.g.: 
https://github.com/OpenMDAO/POEMs/blob/master/POEM_058.md?plain=1#L121